### PR TITLE
Update misc_controls_transform.html

### PR DESCRIPTION
**Pull request by jonTrent**
See related proposed change to TransformControls.js.

### CODE REVIEW

File: examples/misc_controls_transform.html [hunk 1]
```diff
@@ -113,6 +116,35 @@
...
+							break;
+														
+						case 65:
+						case 71:
+						case 88:
+						case 89:
+						case 90: // a, g, x, y, z
+							var gizmoType = control.getMode();
+
+							if ( altMode ) {
```
Comment (mrdoob) [inline examples/misc_controls_transform.html]: I think you can just do `if ( event.altKey ) {`.

**Comment by jonTrent** [comment]
Right you are concerning the simpler way of handling the check for the Alt key being down.  Will simplify the change request accordingly.

File: examples/misc_controls_transform.html [hunk 2]
```diff
@@ -113,6 +114,35 @@
 							control.setSize( Math.max( control.size - 0.1, 0.1 ) );
 							break;
 
+						case 18: // Alt
+							altMode = true;
```
Comment (mrdoob) [inline examples/misc_controls_transform.html]: Is this still needed?

**Comment by jonTrent** [comment]
Ugh, sorry about that.  Lingering code from original version removed, now fully incorporating your suggesting on simplifying the check for the Alt key.

**Comment by jonTrent** [comment]
(See comments to associated pull request #12110.)

**Pull request closed without merging**